gc/src/trace.rs

```diff
@@ -365,3 +366,12 @@ unsafe impl<T: Trace> Trace for VecDeque<T> {
         }
     });
 }
+
+impl<'a, T: ToOwned + Trace + ?Sized> Finalize for Cow<'a, T> {}
+unsafe impl<'a, T: ToOwned + Trace + ?Sized> Trace for Cow<'a, T> {
```

Comment:
Still needs fixes to build.

```suggestion
impl<'a, T: ToOwned + ?Sized> Finalize for Cow<'a, T> {}
unsafe impl<'a, T: ToOwned + ?Sized> Trace for Cow<'a, T>
where
    T::Owned: Trace,
{
```

Also, this file is formatted with `rustfmt`, so you should rerun `rustfmt` (which keeps the `use` declarations sorted at the top and adds a missing trailing newline).